lib/widgets/quantity_selector.dart

```diff
@@ -54,13 +56,26 @@ class _QuantitySelectorState extends State<QuantitySelector> {
             _buildQuantityDisplay(),
             _buildDecreaseQuantityButton(),
           ];
-
+    List<Widget> widgetList2 = new List<Widget>();
+    widgetList2.add(
+      Row(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: widgetList,
+      ),
+    );
+    if (itemsCounterValue == 10)
+      widgetList2.add(Text(
+        stopQuantityText,
+        style: TextStyle(
+          color: paletteForegroundColor,
+          fontSize: 0.4 * widget.iconSize,
+        ),
+      ));
     return Padding(
       padding: EdgeInsets.all(0),
       child: widget.orientation == Orientation.landscape
-          ? Row(
```

Comment:
This Row here is meant for changing the direction of the buttons when the widget.orientation is changed. Since the new text is always placed below the selector widget, I suggest you to add a column at the Padding level. 

```
Padding
  child: Row(
    children: [
      widget.orientation == Orientation.landscape ? Row() : Column(),
      itemsCounterValue == 10 ? Text("Out of Stock") : Container()
```

Reply:
I implemented your suggestions and I moved the rendering logic related to landscape outside the return section to have better code readability. Tell me if you have comments on this. Also, I'm confused with the text under quantity sector and product price.

Reply:
I think your implementation is correct! Can you post the updated video as well? Also, we can do a quick call where I can explain how the text should be displayed!